### PR TITLE
Code changes for complete flow of deployment of MAS Core or MAS Core+Manage and uninstall of deployment..

### PR DESCRIPTION
**Pull request by NatarajBTI**
Code changes for complete flow of deployment of MAS Core or MAS Core+Manage and uninstall of deployment when Undeploy is clicked on UI.
/run pipeline

### CODE REVIEW

File: chart/deploy-mas/templates/02-uninstall-pre-delete.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+kind: Job
+metadata:
+  name: mas-undeploy-sync-role
+  generateName: "mas-undeploy-sync-role-"
+  namespace: mas-uninstall-pipelines
+  annotations:
+    "helm.sh/hook": "pre-delete"
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+    "helm.sh/hook-weight": "10"
+    checkov.io/skip1: CKV_K8S_20
```
Comment (padmankosalaram) [inline chart/deploy-mas/templates/02-uninstall-pre-delete.yaml]: @NatarajBTI Please check if we can fix these checkov violation if possible

**Review by padmankosalaram — CHANGES_REQUESTED** [review]

File: chart/deploy-mas/templates/02-uninstall-pre-delete.yaml (hunk 1 above)
Comment (padmankosalaram) [reply chart/deploy-mas/templates/02-uninstall-pre-delete.yaml]: @NatarajBTI build pipeline has failure, please check

**Comment by NatarajBTI** [comment]
Code was checked in & merged in this PR --> https://github.com/terraform-ibm-modules/terraform-ibm-mas/pull/43. Hence closing this PR.

**Pull request closed without merging**